spec/factories/profiles.rb

```diff
@@ -8,6 +8,13 @@
 
     trait :active do
       active { true }
+      # `activated_at` will be always defined for active profiles,
+      # but it is not defined here so tests will not rely on it
+      # instead of the `active` boolean above as they should
+    end
+
+    trait :deactivated do
+      active { false }
       activated_at { Time.zone.now }
```

Comment:
If it's deactivated, `activated_at` should be in the past, and this should also have a matching `verified_at`.